### DISCUSSION TITLE
GIS open source solution for archaeological context in between Universities, Public Administration, societies, research center: pyArchInit case

### DISCUSSION THREAD

**Discussion by ddomizia** [archeofoss/archeofoss2022#5] [Panel proposals]
Panel proposal

## GIS open source solution for archaeological context in between Universities, Public Administration, societies, research center: pyArchInit case.

![DSC05208](https://user-images.githubusercontent.com/39925492/160680652-a8601f2b-3581-4429-8f30-f2abc3ddf54c.jpg)

Proponents:
- **Enzo Cocca**
  - AdArte s.r.l.
  - [archeologia@adarteinfo.it](mailto:archeologia@adarteinfo.it)
  - [@pyarchinit](https://mobile.twitter.com/pyarchinit)
- **Gianluca Martinez**
  - Università di Pisa, Pisa
  - [gianlucamartinez@hotmail.it](mailto:gianlucamartinez@hotmail.it)

The plugin pyArchInit for QGIS, a GIS tool for archaeological data-management, has been tested in years over two thousand
archaeological excavations by adArte company, in archaeological evaluation and emergency excavations. In the last few years, with the widespread of QGIS, the plugin has been implemented to answer numerous asks from PA and Universities, highlighting some aspects that must be spoken together: in our company database we collected more than hundred thousands of SU, still researchable over ten years, demonstrating the efficiency of the method and data’s durability, with a minimum information loss. During the time, PA receiving the documentation and asking for a development, has demonstrated the usefulness of having an
opensource product, modifiable on needs; Universities that started using it had shown some critical aspect for scientific purpose, leading into the project some radical innovations. We assert that time has come for a common rethinking, stating from to project that have recently adopted the most enhanced usage of the plugin: the San Sisto project, the urban excavation of San Sisto’s church, in the center of Pisa and the excavation of the roman villa “dei Vetti”, in Capraia e Limite (FI), carried on by “Laboratorio di Archeologia medievale” of Pisa’s University and the urban context of the roman theater and “ex filanda” in Fano (PU), carried on by adArte s.r.l. under the supervisory of “Soprintendenza Archeologia, Belle Arti e Paesaggio per le province di Ancona e Pesaro e Urbino”. These two case-study, meanwhile the nature of the actors (Universities, PA, companies), are the starting point for an entire rethinking of the effective usage, possibilities and limits of the plugin, which hopefully will lead to a better development of the product.

Bibliography:

- Cocca E., ANALISI STATISTICHE E GEOSTATISTICHE CON PYARCHINIT: PRIMA SPERIMENTAZIONE, 2016
- Mandolesi L., pyArchInit-python, QGIS e PostgreSQL per la gestione dei dati di scavo, 2012
- Cocca E. 2015, Il GIS nell’ambito di sistemi innovativi per la gestione del dato archeologico. Sviluppo e implementazione di un sistema gestionale e analitico con strumenti open source di banche dati archeologiche. Caso studio Grotta di Fumane, Tesi
di Dottorato di ricerca in Scienze e Tecnologie per l’Archeologia e i Beni Culturali, Università di Ferrara, XXV ciclo, a.a. 2015.
- Mandolesi L., Cocca E. 2013, Pyarchinit: gli sviluppi dopo ArcheoFOSS 2009, in M. Ser-lorenzi (ed.), ArcheoFOSS. Free, Libre and Open Source Software e Open Format nei processi di ricerca archeologica. Atti del VII Workshop (Roma 2012), «Archeologia e
Calcolatori», Suppl. 4, 128-138.
- Mandolesi L, Montagnetti R., QGIS, pyarchinit and blender: Surveying and Management of Archaeological data with Open Source Solutions, 2020

**Comment by pyarchinit** [comment]
Chiedo una modifica di due refusi dovuti a fretta e a varie riscritture:

all'inizio è rimasto un QIS al posto di QGIS

in fondo va eliminato il 'si chiama' tra Soprintendenza e Archeologia

Grazie

**Reply by jbogdani** [reply]
Grazie per la segnalazione. Tutto corretto.
